### DISCUSSION TITLE
English UI

### DISCUSSION THREAD

**Discussion by Syed1337** [CopyPlusPlus/CopyPlusPlus-NetFramework#50] [Ideas]
Your app is awesome; please make its interface in English.

**Comment by wy-luke** [comment]
@Syed1337 Thank you so much for your appreciation! English UI is in proress, and a few days later it will be done.
Btw, I wonder where did you find my app?

**Reply by Syed1337** [reply]
Great news. A friend posted about your app in a WeChat group.

**Comment by Syed1337** [comment]
Please also add an option for custom shortcut instead of "`Ctrl +c+c`". So we could use it with elgato stream deck.

**Reply by wy-luke** [reply]
Yeah, the new version will have that option.

**Comment by wy-luke** [comment]
@Syed1337 I have wrote a initial version, would you like to try and test it?

**Comment by SRR8** [comment]
Are you still working on this? Last update was in 2022

**Reply by wy-luke** [reply]
Currently not, but maybe I will restart doing it soon. Thanks for your support~
